core/controllers/admin.py

```diff
@@ -369,3 +369,12 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class AdminConfig(base.BaseHandler):
```

Comment:
Could we actually put this handler in reader.py and restrict it to getting only the playthrough config? Don't want to expose arbitrary admin config properties, just the playthrough one.

The existing previous code in reader.py is also a good pattern that could perhaps be used for editor.py similarly.